kclvm/sema/src/ty/context.rs

```diff
@@ -52,6 +55,27 @@ impl TypeContext {
         is_cyclic_directed(&self.dep_graph)
     }
 
+    /// Add dependencies and return the dependency graph is cyclic.
+    pub fn add_dependencies(&mut self, from: &str, to: &str) -> bool {
```

Comment:
the bool result may lead to ambiguity: add succeed or failed?

Reply:
> the bool result may lead to ambiguity: add succeed or failed?

I will fix it on another PR.

Reply:
> the bool result may lead to ambiguity: add succeed or failed?

fixed.